frontend/cypress/e2e/createEvent.cy.ts

```diff
@@ -0,0 +1,16 @@
+describe('template spec', () => {
+    it('passes', () => {
+        cy.visit('/');
+        cy.get("[role=gridcell]").first().click();
+        cy.contains("New event").click();
+
+        cy.get("[value=\"New event\"]").clear().type("test_event");
+        cy.contains("Save").click();
+
+        cy.get("[class=\"fc-event-title fc-sticky\"]").contains("test_event");
+
+        cy.contains("Delete").click();
+
+        cy.get("[class=\"fc-event-title fc-sticky\"]").contains("test_event").should("not.exist");
```

Comment:
it's better to select elements not by class but by unique unchangeable id. usually in cypress it `data-cy` field. in this case you should add such field to required elements
and select like cy.getByCyId('modal-title');